JSDemos/Demos/PivotGrid/ChartIntegration/Vue/description.md

```diff
@@ -1 +1,44 @@
-DevExtreme Vue Pivot Grid is&nbsp;a&nbsp;powerful client-side component for multi-dimensional data analysis. It&nbsp;provides such essential features as&nbsp;grouping, sorting, filtering, multiple summary calculation modes, flexible layout, and more. You can use Vue syntax and techniques to&nbsp;instantiate and configure the Pivot Grid or&nbsp;handle its events. [Find out more about DevExtreme Vue components](/Documentation/Guide/Vue_Components/DevExtreme_Vue_Components/).
\ No newline at end of file
+PivotGrid allows you to integrate the [Chart](/Documentation/ApiReference/UI_Components/dxChart/) component to visualize data. The Chart is synchronized with the PivotGrid and visualizes the currently displayed total summary values. Expand a row or column header in the PivotGrid below to see how it works.
+
+## Bind the Chart to the PivotGrid
+To bind these two components, create a Chart and pass its instance as the first argument to the [bindChart(chart, integrationOptions)](/Documentation/ApiReference/UI_Components/dxPivotGrid/Methods/#bindChartchart_integrationOptions) method. You can call this method at any point in your application. In this demo, we call this method in the <a href="https://v3.vuejs.org/api/options-lifecycle-hooks.html#mounted" target="_blank">mounted</a> and use instances to bind the components.
```

Comment:
`In this demo, we call this method in the <a href="https://v3.vuejs.org/api/options-lifecycle-hooks.html#mounted" target="_blank">mounted</a> and use instances to bind the components.`

Can we specify what "mounted" is (for example, an option, a function)? For example, can we say "in the mounted option" or "in the mounted function"?